cmd/influxd/run/config.go

```diff
@@ -69,8 +69,10 @@ func NewConfig() *Config {
 	c.Monitor = monitor.NewConfig()
 	c.Subscriber = subscriber.NewConfig()
 	c.HTTPD = httpd.NewConfig()
+	c.Graphites = []graphite.Config{graphite.NewConfig()}
```

Comment:
These should probably be added in the `NewDemoConfig`, not the `NewConfig` section.  `NewConfig` is purposefully a minimal set of defaults that will be set to run that merges with the config passed to it.  Putting them here would enable graphite/udp for everyone even if they didn't specify it.